src/main/java/org/jabref/gui/preferences/general/GeneralTabViewModel.java

```diff
@@ -125,8 +129,18 @@ public void setValues() {
         // The light theme is in fact the absence of any theme modifying 'base.css'. Another embedded theme like
         // 'dark.css', stored in the classpath, can be introduced in {@link org.jabref.gui.theme.Theme}.
         switch (workspacePreferences.getTheme().getType()) {
-            case DEFAULT -> selectedThemeProperty.setValue(ThemeTypes.LIGHT);
-            case EMBEDDED -> selectedThemeProperty.setValue(ThemeTypes.DARK);
+            case DEFAULT -> {
+                selectedThemeProperty.setValue(ThemeTypes.LIGHT);
+                if (workspacePreferences.automaticThemeDetectionFlag().getValue()) {
+                    selectedThemeProperty.setValue(ThemeTypes.SYNC);
+                }
+            }
+            case EMBEDDED -> {
+                selectedThemeProperty.setValue(ThemeTypes.DARK);
```

Comment:
I don't get, why first DARK is set and then SYNC. Why not an `else` branch at the if?